lib/nerves_runtime/init.ex

```diff
@@ -155,4 +156,12 @@ defmodule Nerves.Runtime.Init do
   end
 
   defp validate_mount(s), do: s.mounted
+
+  defp maybe_validate_fw() do
+    validated? = KV.get("nerves_fw_validated") == "1"
+
+    if Application.get_env(:nerves_runtime, :autovalidate, true) && not validated? do
```

Comment:
Perhaps `:validate_firmware` for the key?

I'm feeling that my use of the word `autovalidate` in the past was not the best choice, and since we have an opportunity to revisit the words that we might as well try to find something better.

Reply:
I like this key. Updated